src/index.html

```diff
@@ -492,7 +492,7 @@ <h1>Contact</h1>
 				<p>Nu ezita să ne adresezi orice întrebare sau comentariu, prin mail sau pe pagina de facebook.</p>
 				<div>
 					<a class="contact-link" href="https://www.facebook.com/asmi.ub/" target="_blank"><i class="fa fa-facebook-official" aria-hidden="true"></i>facebook.com/asmi.ub</a>
-					<a class="contact-link" href="mailto:asmi.unibuc@gmail.com"><i class="fa fa-envelope" aria-hidden="true"></i>asmi.unibuc@gmail.com</a>
+					<a class="contact-link" href="mailto:cd@asmi.ro"><i class="fa fa-envelope" aria-hidden="true"></i>cd@asmi.ro</a>
```

Comment:
```suggestion
					<a class="contact-link" href="mailto:smarthack@asmi.ro"><i class="fa fa-envelope" aria-hidden="true"></i>smarthack@asmi.ro</a>
```
Merge sugestia asta? Ar trebui să poți da direct commit din GH :D